examples/python_editor.py

```diff
@@ -133,15 +91,21 @@ def __init__(self, **traits):
                 ),
                 Group(
                     Action(
-                        name="Show Line Numbers",
+                        name="Lines",
                         style='toggle',
                         on_perform=self.on_show_line_numbers,
                         checked=True,
                     ),
-                    IntFieldAction(
-                        name="Go to line",
-                        on_perform=self.on_go_to_line,
-                    )
+                    FieldAction(
+                        name='Style',
+                        field_type=ComboField,
+                        field_defaults={
+                            'values': styles,
+                            'value': u'default',
+                            'tooltip': u'Style',
```

Comment:
It's probably safe to import `unicode_literals` here instead of using `u`-prefixes, right?